mgmtsystem/hawkular.py

```diff
@@ -137,47 +137,55 @@ def list_server_deployment(self, feed_id, type_id='Deployment'):
         entities = []
         entities_j = self.api.get_json('feeds/{}/resourceTypes/{}/resources'
                                        .format(feed_id, type_id))
-        for entity_j in entities_j:
-            entity = Deployment(entity_j['id'], entity_j['name'], entity_j['path'])
-            entities.append(entity)
+        if entities_j is not None:
+            for entity_j in entities_j:
+                entity = Deployment(entity_j['id'], entity_j['name'], entity_j['path'])
+                entities.append(entity)
         return entities
 
     def list_feed(self):
         """Returns list of feeds"""
         entities = []
         entities_j = self.api.get_json('feeds')
-        for entity_j in entities_j:
-            entity = Feed(entity_j['id'],
+        if entities_j is not None:
+            for entity_j in entities_j:
+                entity = Feed(entity_j['id'],
                           entity_j['name'] if entity_j.__contains__('name') else None,
```

Comment:
Can you also change `entity_j.__contains__('name')` to `'name' in entity_j` ? [Explanation](http://stackoverflow.com/a/1964949)